crates/matrix-sdk-common/src/deserialized_responses.rs

```diff
@@ -423,6 +444,35 @@ impl fmt::Debug for TimelineEvent {
     }
 }
 
+#[derive(Clone, Serialize, Deserialize)]
+/// A successfully-decrypted encrypted event.
```

Comment:
The casual reader might wonder why this is here, rather than in the crypto crate. The answer is that, in the fullness of time, I want to make `TimelineEvent` and `SyncTimelineEvent` wrap a `DecryptedRoomEvent` rather than duplicating the fields.